docs/actions-cohortextractor.md

```diff
@@ -56,13 +56,15 @@ A basic `generate_cohort` action looks like this:
 
 ```yaml
 generate_study_cohort
-  run: cohortextractor:latest generate_cohort --output-format=csv.gz
+  run: cohortextractor:latest generate_cohort --output-format =csv.gz
```

Comment:
I think you can either use a space or an equals here, but I'm not sure about both. I think it should be either:
```
--output-format=csv.gz
```
or
```
--output-format csv.gz
```

Reply:
Oops thanks